app/controllers/carts_controller.rb

```diff
@@ -1,11 +1,13 @@
 class CartsController < ApplicationController
-  before_action :set_cart, only: [:show, :edit, :update, :destroy]
+  include CurrentCart
```

Comment:
done. kept set_cart in different controllers as its 'only-s' are different in different controllers.